app/scripts/requirements

```diff
@@ -0,0 +1,96 @@
+#!/usr/bin/env php
+<?php
+
+// Script should not be called from the Web
+if (php_sapi_name() != 'cli') {
+    die('Nope');
+}
+
+function colorizeOutput($text, $color)
+    {
+        switch ($color) {
+            case 'green':
+                // White on green
+                $output = "\033[1;37m\033[42m{$text}";
+                break;
+            case 'red':
+                // White on red
+                $output = "\033[1;37m\033[41m{$text}";
+                break;
+            default:
+                $output = $text;
+                break;
+        }
+        $output .= "\033[0m";
+        return $output;
+    }
+
+/**
+ * Checks a configuration.
+ */
+function check($boolean, $message, $help = '', $fatal = false, $fix = '')
+{
+    $text = sprintf("* $message%s\n", $boolean ? '' : ': FAILED');
+
+    if (! $boolean) {
+        if(! empty($help)) { 
+            $text .=  "  $help\n";
+        }
+        if(! empty($fix)) {
+                $text .=  "  $fix\n";
+        }
+    }
+    if($boolean) {
+        echo colorizeOutput($text, 'green');
+    } else {
+        echo colorizeOutput($text, 'red');
+    }
+    if (! $boolean && $fatal)
+    {
+        die("You must fix this problem before resuming the check.\n");
+    }
+}
+
+function isEmptyDir($dir)
+{
+        if (($files = @scandir($dir)) && count($files) <= 2) {
+                return true;
+        }
+        
+        return false;
+}
+
+echo " Transvision Requirements Check\n";
+
+// get the parent folder from the script itself
+$root = dirname(dirname(dirname(__FILE__)));
+
+// load settings and constants
+$server_config = parse_ini_file($root . '/app/config/config.ini');
+include_once($root . '/app/inc/constants.php');
+
+// check php version
+check(version_compare(phpversion(), '5.4', '>='), sprintf('PHP version is at least 5.4 (%s)', phpversion()), 'Current version is '.phpversion(), true);
```

Comment:
Leave an empty space before and after operators, `'Current version is ' . phpversion()`.
You should also store `phpversion()` value instead of calling it several times.